### ISSUE TITLE
when you have the subscription for the WCAG levvels remove the greyout and lock icon

### ISSUE THREAD

**Issue by Ethan-Kruger** [Ethan-Kruger/ADA-Checker#68]
- [ ] remove lock icon
- [ ] replace grey text with black when you have the desired subscription
